README.md

```diff
@@ -155,65 +155,6 @@ Ably depends on our [MessagePack Fork](https://github.com/ably-forks/msgpack-obj
 
 ---
 
-## Push Notifications
-
-If you haven’t yet, you should first check the detailed [documentation](https://www.ably.com/docs/general/push). An [example app for push notifications](https://github.com/ably/push-example-ios) is also available.
-
-### Activation and device registration
-
-For more information, see [Push Notifications - Device activation and subscription](https://ably.com/docs/general/push/activate-subscribe).
-
-`ARTPushRegistererDelegate` defines 4 delegate methods to handle the outcome of push activation, deactivation and update events. By default, the Ably SDK will check if `UIApplication.sharedApplication.delegate` conforms to `ARTPushRegistererDelegate`, and call the delegate methods when appropriate. Therefore, specifying the `ARTPushRegistererDelegate` is optional. To use a different class implementing `ARTPushRegistererDelegate`, you must provide this class to Ably, by setting the `ARTClientOptions.pushRegistererDelegate` delegate. In SwiftUI applications, you must set the `ARTClientOptions.pushRegistererDelegate` delegate property.
-
-Do not forget that `ARTPush` has two corresponding methods that you should call from yours [application(_:didRegisterForRemoteNotificationsWithDeviceToken:)](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622958-application) and [application(_:didFailToRegisterForRemoteNotificationsWithError:)](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622962-application), passing to them also an `ARTRest` or `ARTRealtime` instance, configured with the authentication setup and other options you need:
-
-```
-func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
-    ARTPush.didRegisterForRemoteNotifications(withDeviceToken: deviceToken, rest: rest)
-}
-
-func application(_ application: UIApplication, didFailToRegisterForRemoteNotificationsWithError error: Error) {
-    ARTPush.didFailToRegisterForRemoteNotificationsWithError(error, rest: rest)
-}
-```
-
-Only one instance of `ARTRest` or `ARTRealtime` at a time must be [activated for receiving push notifications](https://www.ably.com/docs/general/push/activate-subscribe). Having more than one activated instance at a time may have unexpected consequences.
-
-### Location pushes
```

Comment:
Yeah, as you mentioned in the PR description, I think that this is the only place that we document location pushes, so we need to make sure this documentation exists somewhere before merging.